src/core/settings/qgssettingsregistrycore.h

```diff
@@ -173,6 +173,9 @@ class CORE_EXPORT QgsSettingsRegistryCore : public QgsSettingsRegistry
 
     //! Settings entry show feature counts for newly added layers by default
     static const inline QgsSettingsEntryBool settingsLayerTreeShowFeatureCountForNewLayers = QgsSettingsEntryBool( QStringLiteral( "show_feature_count_for_new_layers" ), QgsSettings::Prefix::CORE_LAYERTREE, false, QStringLiteral( "If true, feature counts will be shown in the layer tree for all newly added layers." ) );
+
+    //! Settings entry enable WMS tile prefetching.
+    static const inline QgsSettingsEntryBool settingsEnableWMSTilePrefetching = QgsSettingsEntryBool( QStringLiteral( "enableWmsTilePrefetch" ), QgsSettings::Prefix::QGIS, false );
```

Comment:
```suggestion
    static const inline QgsSettingsEntryBool settingsEnableWMSTilePrefetching = QgsSettingsEntryBool( QStringLiteral( "enable_wms_tile_prefetch" ), QgsSettings::Prefix::QGIS, false );
```

I'd also suggest creating a new prefix for WMS here instead of using the QGIS generic one

Reply:
Thanks for the review, I've applied the suggested changes